src/extensions/giphyComponent.js

```diff
@@ -220,7 +220,6 @@ GiphyComponent.prototype.render = function(element, options) {
       this.selectionDom = document.createElement('div');
       this.selectionDom.innerHTML = '&nbsp;';
       this.selectionDom.className = 'selection-pointer';
-      this.selectionDom.setAttribute('contenteditable', true);
```

Comment:
This change is breaking the selection process of the elements (including `FigureComponent`). 

Try for example clicking on an image and notice that there's a cursor in the paragraph before hitting delete won't delete the image and instead would delete the character in previous paragraph.